src/read.c

```diff
@@ -2879,6 +2879,10 @@ static avifResult avifParseItemReferenceBox(avifMeta * meta, const uint8_t * raw
 
     uint8_t version;
     AVIF_CHECKERR(avifROStreamReadVersionAndFlags(&s, &version, NULL), AVIF_RESULT_BMFF_PARSE_FAILED);
+    if (version > 1) {
+        // iref versions > 1 are not supported. skip it.
```

Comment:
Nit: Capitalize "skip" because it is after a period.

Reply:
Done.